patterns/1-initial/extensions-for-sustainable-growth.md

```diff
@@ -8,26 +8,26 @@ An InnerSource project is receiving too many contributions, making maintenance d
 
```

Comment:
Putting it here because I can't add it on the picture directly - do you know how diagrams are embedded into the book in general? Mostly wondering if this should be directly embedded in the text, with the caption in text as well

Reply:
great question, I actually followed one of the other structured patterns. I am not sure how it goes inside the book, but I assume they have some publication process and read from assets/images